src/state/queries/notifications/util.ts

```diff
@@ -205,8 +206,15 @@ function getSubjectUri(
   type: NotificationType,
   notif: AppBskyNotificationListNotifications.Notification,
 ): string | undefined {
-  if (type === 'reply' || type === 'quote' || type === 'mention') {
+  if (type === 'reply' || type === 'mention') {
     return notif.uri
+  } else if (type === 'quote') {
+    if (
+      AppBskyFeedPost.isRecord(notif.record) &&
+      AppBskyEmbedRecord.isMain(notif.record.embed)
+    ) {
+      return notif.record.embed.record.uri
+    }
```

Comment:
Shouldn't the subject be the quoted post's URI here? A quote post will never be muted since it's new, but we might have the quoted muted.

Reply:
Ah no -- the subject should be the quote itself. The subject is the content that drove the notification; it drives what will show in the notifs. If we return the embed target (the quoted post) then the notification would render as your own post

Reply:
Right, good catch thanks. Changed logic to just check the embed inside the `isThreadMuted` codepath the same way we would without the subject. Only applies to the root, of course